bot/exts/evergreen/tic_tac_toe.py

```diff
@@ -84,7 +84,7 @@ async def get_move(self, board: t.Dict[int, str], _: discord.Message) -> t.Tuple
                 board_copy = board.copy()
                 board_copy[move] = symbol
                 if check_win(board_copy):
-                    return False, move
+                    return (False, move)
```

Comment:
I don't feel like this change is necessary. If two items are returned at the same time, it will be returned as a tuple of those two items.

Reply:
_i just added to for no real reason :)_

Reply:
Then I'm removing this change.